packages/test/snapshots/src/test/replay.spec.ts

```diff
@@ -3,24 +3,34 @@
  * Licensed under the MIT License.
  */
 
-import { Mode, processContent } from "../replayMultipleFiles";
+import { Mode, processContent, testCollateralExists } from "../replayMultipleFiles";
 
 describe("Snapshots", function() {
     this.timeout(300000);
 
-    it("Stress Test", async () => {
+    function skipIfCollateralDoesNotExist(itFnScope) {
+        if (!testCollateralExists()) {
+            itFnScope.skip();
+        }
+    }
+
+    it("Stress Test", async function() {
+        skipIfCollateralDoesNotExist(this);
```

Comment:
can this go in a beforeEach rather than in every test

Reply:
Yes! I knew this was bothering me for some reason and `beforeEach` didn't immediately come to mind. I'll make the change.